torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1602,6 +1603,24 @@ const std::vector<uint64_t>& ProcessGroupNCCL::groupRanks() const {
   return options_->global_ranks_in_group;
 }
 
+void ProcessGroupNCCL::setEphemeralTimeout(
```

Comment:
Wouldn't add be better than set, as you are accumulating ephemeral timeout here?